### ISSUE TITLE
DOSXYZ compilation error

### ISSUE THREAD

**Issue by MrSlon** [nrc-cnrc/EGSnrc#597]
Trying to run DOSXYZnrc under Linux using input files that previously worked. Compiler returns the following error:

![Error](https://user-images.githubusercontent.com/62914849/81770991-6f8a3800-94a7-11ea-9bfa-15197a8856a6.png)

What could that be and how to fix it? Thanks!

**Comment by rtownson** [comment]
Is this a 2015 version of EGSnrc? Have you changed something since then that might have caused it to stop working (did it work previously)? One option would be to try EGSnrc 2020 instead.

**Comment by MrSlon** [comment]
Yes, it's a 2015 version of EGS installed in our institutional Linux cluster. It can process calculations much faster. I'm not sure it would be possible to re-install another version, but will ask.
Yes, previously it worked well. Also, input files I used worked on another computer, so they are good. Just unsure why this error pops up...

**Comment by crcrewso** [comment]
I think this was the issue with previous EGSnrc compile stacks where IAEA libraries wouldn't be compiled #49 











thi

**Comment by rtownson** [comment]
Does your DOSXYZnrc Makefile contain this line?
```
$(EGS_UTILS)phsp_macros.mortran $(IAEA_PHSP_MACROS)\
```
If it's missing the IAEA_PHSP_MACROS part, add it and try recompiling.
